documentation/slate/source/includes/examples.md

```diff
@@ -196,12 +196,13 @@ a journey can be faster than an other but requires more changes or more walking.
 This API has more options explained in the reference as:
 
 * The dates are given in the basic form of the ISO 8601 datetime format: ``YYYYMMDDTHHMM``.
-  For example, if you want to compute a journey on friday, April 22 ``datetime=20160422T0800``
-  <https://api.navitia.io/v1/journeys?from=-122.47787733594924;37.71696489300146&to=-122.41539259473815;37.78564348914185&datetime=20160422T0800>
+  For example, if you want to compute a journey on friday, April 07 ``datetime=20170407T120000``
+  <https://api.navitia.io/v1/journeys?from=-122.47787733594924;37.71696489300146&to=-122.41539259473815;37.78564348914185&datetime=20170407T120000>
+
 
 * Latest departure
   To get the latest departure, you can query for some journeys arriving before the end of the service
-  <https://api.navitia.io/v1/journeys?from=-122.47787733594924;37.71696489300146&to=-122.41539259473815;37.78564348914185&datetime=20160422T2359&datetime_represents=arrival>
+  <https://api.navitia.io/v1/journeys?from=-122.47787733594924;37.71696489300146&to=-122.41539259473815;37.78564348914185&datetime=20170407T120000&datetime_represents=arrival>
```

Comment:
the sandbox OSM data are shit, there is no valid autocomplete.

We should update the osm data and update all documentation examples to point to it, but I don't think that's in the scope of this PR